stimela/backends/kube/daskjob.py

```diff
@@ -177,14 +177,14 @@ def daskjob_template(args):
                                         "$(DASK_WORKER_NAME)",
                                         "--nworkers", '1',
                                         "--nthreads", str(args.threads_per_worker),
-                                        "--memory-limit", str(args.memory_limit),
+                                        "--memory-limit", str(0),  #str(args.memory_limit),
```

Comment:
Why's this?

Reply:
Oh sorry, forgot about that. Dask will start spilling to disk at the stupidly configured 60% default without it. I guess we need a way to to set this in kubeconfig instead of hardcoding it. Actually it would be great if there was a way to expose more of the cluster config in kubeconfig